doc/engineering-reference/src/integrated-solution-manager/zone-air-mass-flow-conservation.tex

```diff
@@ -54,12 +54,17 @@ \subsubsection{Zone Return Air Node 1}\label{zone-return-air-node1}
 {\dot m_{R,1}} = ReturnFlowSchedule*{\dot m_{R}}
 \end{equation}
 
-The zone air mass flow conservation calculation limits the zone the return node air flow rate with air loop design supply to protect from large number the solution may throw as follows:
+In the zone air mass flow conservation calculation, the calculated zone total return flow rate is modified using return node flow schedule value when the zone air balancing method is AdjustMixingOnly and assigned to return node 1.  
 
 \begin{equation}
-{\dot m_{R,1}} = MIN\left( {{\dot m_{R,1}},\,{{\dot m}_{DesSupply,i}}} \right) 
+{\dot m_{R,1}} = ReturnFlowSchedule*{\dot m_{R}}
 \end{equation}
```

Comment:
Edited and rearranged the flow and updated how the return flow schedule multiplier is used for a zone with single return node.